tests/integration/fee-estimation.test.ts

```diff
@@ -38,15 +38,15 @@ describe('fee estimation tests', async () => {
     .addChain({
       identity: {
         name: 'Relay',
-        universalLocation: relaychainUniversalLocation('westend'),
+        universalLocation: relaychainUniversalLocation('kusama'),
```

Comment:
It's better to use Westend's `ByGenesis` here. Kusama has too long timings.

Maybe it's even worth adding the `westendRelayNetworkId` function.